tests/functional/aws-node-sdk/test/versioning/objectCopy.js

```diff
@@ -684,7 +687,7 @@ testing('Object Version Copy', () => {
                     Key: sourceObjName, ACL: 'public-read' }, () => {
                     otherAccountS3.copyObject({ Bucket: otherAccountBucket,
                         Key: otherAccountKey,
-                        CopySource: copySource,
+                        CopySource: `${sourceBucketName}/${sourceObjName}`,
```

Comment:
in `                 s3.putObjectAcl({ Bucket: sourceBucketName,
                      Key: sourceObjName, ACL: 'public-read' }`
can you change to 
```
                 s3.putObjectAcl({ Bucket: sourceBucketName,
                      Key: sourceObjName, ACL: 'public-read', VersionId: versionId }
```
?
so you can leave the copy source with the version id